game-app/game-core/src/main/java/games/strategy/engine/framework/map/file/system/loader/DownloadedMapsListing.java

```diff
@@ -35,8 +36,8 @@ public static synchronized DownloadedMapsListing parseMapFiles() {
 
   private static Collection<DownloadedMap> readMapYamlsAndGenerateMissingMapYamls() {
     // loop over all maps, find and parse a 'map.yml' file, if not found attempt to generate it
-    return FileUtils.listFiles(ClientFileSystemHelper.getUserMapsFolder()).stream()
-        .filter(File::isDirectory)
+    return FileUtils.listFiles(ClientFileSystemHelper.getUserMapsFolder().toPath()).stream()
```

Comment:
Why not have `ClientFileSystemHelper.getUserMapsFolder()` return a path? Is that an upcoming change?

Reply:
Yes it is. `ClientFileSystemHelper.getUserMapsFolder()` has more than 10+ usages when I checked and that's simply too much for this PR